### DISCUSSION TITLE
After login to my account this page appear

### DISCUSSION THREAD

**Discussion by TAHIDUR** [community/community#33128] [Other Feature Feedback, Questions, & Ideas]
![image](https://user-images.githubusercontent.com/25381402/190318863-6ed68cc5-7d0a-4228-a5f0-fc60710bb1bb.png)

**Comment by xiaomi7732** [comment]
A workaround: try to go to `www.github.com/<youraccountname>` instead.

**Comment by haquenafeem** [comment]
try github.com/{username}
